app/com/gu/floodgate/AppComponents.scala

```diff
@@ -63,11 +64,23 @@ class AppComponents(context: Context) extends BuiltInComponentsFromContext(conte
   )
 
   val reindexService = new ReindexService(contentSourceService, runningJobService, jobHistoryService, progressTrackerControllerActor, wsClient)
-
-  val contentSourceController = new ContentSourceApi(contentSourceService, reindexService, jobHistoryService, runningJobService)
   val runningJobController = new RunningJobApi(runningJobService)
   val jobHistoryController = new JobHistoryApi(jobHistoryService)
 
+  val previewCodeBulkJobActor = generateBulkActors("draft-code")
```

Comment:
Thinking out loud here: I don't know how lightweight actors are, but wouldn't it be more efficient if the environment was just a value you pass around at the calling site?